packages/core-sdk/resolvers/account-resolvers/interface/query.graphql

```diff
@@ -1,72 +1,130 @@
+  account: String!
+  chainId: String!
+  tokenBalances: [TokenBalance!]!
 }
 
 type TokenBalance {
   token: TokenResolver_Token!
   balance: String!
 }
 
-# TODO
-# enum TransferType {
-#   IN,
-#   OUT,
-# }
-
-# type Transfer {
-#   txHash: String!
-#   from: String!
-#   to: String!
-#   value: String!  # BigDecimal
-#   transferType: TransferType!
-#   timestamp: String!
-# }
-
-# type TransfersList {
-#   token: Token!
-#   transfers: [Transfer!]!
-#   pagination: Pagination!
-# }
-
-# type Options {
-#   pagination: PaginationOptions
-#   blockRange: BlockRangeOptions
-# }
-
-# type PaginationOptions {
-#   page: Int!
-#   perPage: Int!
-# }
-
-# type BlockRangeOptions {
-#   startBlock: Int  # default: 0
-#   endBlock: Int  # default: latest
-# }
-
-# type Pagination {
-#   total: Int!
-#   perPage: Int!
-#   page: Int!
-#   hasMore: Boolean!
-# }
-
-# type Transaction {
-#   hash: String!
-#   from: String!
-#   to: String!
-#   value: String!  # BigDecimal
-#   gasUsed: Int!  # Gas used for the transaction
-#   gasLimit: Int!  # Gas offered for the transaction
-#   gasPrice: String!  # Gas price in gwei (BigDecimal)
-#   timestamp: String!  # Timestamp of the transaction
-#   blockHeight: Int!  # Block height of the transaction
-#   offset: Int!
-# }
+type Options {
+  pagination: PaginationOptions
+  blockRange: BlockRangeOptions
+}
+
+type PaginationOptions {
+  page: Int!
+  perPage: Int!
+}
+
+type BlockRangeOptions {
+  startBlock: Int # default: 0
+  endBlock: Int # default: latest
+}
+
+type Pagination {
+  total: Int
+  perPage: Int
+  page: Int
+  hasMore: Boolean
+}
+
+type GasInfo {
+  offered: String! # Gas offered for the transaction
+  spent: String! # Gas spent for the transaction
+  price: String! # Gas price in wei (BigDecimal)
+  quoteRate: String! # rate of native currency in quoted currency (BigDecimal)
+  quote: String! # Gas price in quote currency (BigDecimal)
+}
+
+type Transaction {
+  hash: String!
+  from: String!
+  to: String!
+  value: String! # BigDecimal
+  quote: String! # BigDecimal
+  gasInfo: GasInfo!
+  logs: [EventLog!]!
+  timestamp: String! # Timestamp of the transaction
+  blockHeight: Int! # Block height of the transaction
+  offset: Int!
+}
+
+type TransactionsList {
+  account: String!
+  chainId: String!
+  quoteCurrency: String!
+  transactions: [Transaction!]!
+  pagination: Pagination!
+  updatedAt: String!
+  nextUpdateAt: String!
+}
+
+enum TransferType {
+  IN
+  OUT
+}
+
+type Transfer {
+  from: String!
+  to: String!
+  type: TransferType!
+  value: String! # BigDecimal
+  quoteRate: String! # rate of token in quote currency (BigDecimal)
+  quote: String! # BigDecimal
+}
+
+type TransfersPerTx {
+  transaction: Transaction!
+  transfers: [Transfer!]!
+}
+
+type TransfersList {
+  account: String!
+  chainId: String!
+  quoteCurrency: String!
+  token: TokenResolver_Token!
+  transfers: [TransfersPerTx!]!
+  pagination: Pagination!
+  updatedAt: String!
+  nextUpdateAt: String!
```

Comment:
How does nextUpdateAt work?

Reply:
If this is a bad question because it's part of the API, just let me know. :)

Reply:
It's part of the API and they provide this field to let users know when they will be updating the API data next.

Reply:
Thanks for pointing this out, I think this should be an optional field since not all the APIs would be providing this info.